lib/animation.dart

```diff
@@ -204,6 +209,9 @@ class Animation {
       return;
     }
     if (!loop && isLastFrame) {
+      if (onCompleteAnimation != null) {
```

Comment:
To not do null checks you can do this little neat trick in Dart if you want:
`onCompleteAnimation?.call();`

Reply:
elvis operator in action :P

Reply:
Close, kind of the opposite of an elvis operator. :)
The elvis operator is ?? in dart.